### PR TITLE
Update travis yaml

### PR DESCRIPTION
**Pull request by mintyfloss**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,9 +1,16 @@
 language: node_js
 node_js:
-  - "10"
+  - "10.4.1"
```
Comment (pierceray) [inline .travis.yml]: Did this fail with just 10 there?

Comment (mintyfloss) [reply .travis.yml]: Yes, here's the failing PR https://github.com/RetailMeNotSandbox/roux-resolver-webpack-plugin/pull/18

Comment (mintyfloss) [reply .travis.yml]: Let me include the debug change in here and confirm that build is fine.

**Comment by mintyfloss** [comment]
Issue created https://github.com/RetailMeNotSandbox/roux-resolver-webpack-plugin/issues/20

**Pull request merged**